src/rust/ide/lib/view/src/graph_editor/component/breadcrumbs/breadcrumb.rs

```diff
@@ -0,0 +1,416 @@
+//! This module provides a clickable view for a single breadcrumb.
+
+use crate::prelude::*;
+
+use super::GLYPH_WIDTH;
+use super::HORIZONTAL_MARGIN;
+use super::VERTICAL_MARGIN;
+use super::TEXT_SIZE;
+
```

Comment:
The variable `PADDING` doesn't work. Try setting it as `100.0` or `-100.0` - nothing will change.

Reply:
@wdanilo as mentioned in a previous comment, PADDING only adds a padding to the shape size to make the SDF shapes fit in.

Reply:
@wdanilo I've implemented proper actual padding. Could you please check it?